architecture/core_concepts/projects_and_users.adoc

```diff
@@ -134,3 +134,30 @@ endif::[]
 Developers and administrators can xref:../../dev_guide/projects.adoc#dev-guide-projects[interact
 with projects] using xref:../../cli_reference/index.adoc#cli-reference-index[the CLI] or the
 xref:../infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
+
+ifdef::openshift-online[]
+[[projects-hibenation]]
+=== Project Hibernation
+In {product-title} Starter, a project exceeding 54 cumulative quota-hours of
+usage in a rolling 72-hour period must sleep for the next 18 hours. Also, a
+project that has been inactive for more than 30 minutes will be idled. When a
```

Comment:
@damemi is the 54/72/18 hrs correct? @ahardin-rh We might need to say that's it's 'configured' but then it seems silly to say:
'a project exceeding configured quota-hours limit of usage in a rolling configured time period must sleep for a configured amount of time.'  and 'a project that has been inactive for more than a configured time will be idled'

Reply:
@abhgupta do you know what times we should state for force sleep? After discussion, we don't have a consensus. Thanks!

Reply:
@abhgupta bump

Reply:
@sallyom @abhgupta When are these docs targeted to be published?

Reply:
@ahardin @agupta, currently hibernation is configured 'exceeding 16 cumulative quota-hours of
+usage in a rolling 24-hour period must sleep for the next 8 hours.'  However, forcesleep is still in 'dryRun' in all starter clusters.. only auto-idling is turned on atm

Reply:
@sallyom What does dry run mean to the user? Does that just mean it's optional? If so, what is a use case for it?